amp/TEMPLATE/reamp/modules/activity-preview/actions/CommonListsActions.jsx

```diff
@@ -0,0 +1,175 @@
+import commonListsApi from '../api/CommonListsApi';
+
+
+/**
+ * @author Daniel Oliva
```

Comment:
Since the code is coming from amp, please remove the author and add the file and a link to the GitHub revision from there we took this code so it is traceable when we decide to merge both codes (preview from Offline and and online preview)

Reply:
The classnames have the same name as amponline (and most of the folder structure remains), but I couldn't use the code from it since it has lot of bugs and the endpoints that both apps consumes are different. Good luck if you try to merge the code, I will remove my name so nobody could find me at such moment.